cypress/integration/buy-shirt.spec.ts

```diff
@@ -17,46 +17,28 @@ const paymentStepPage = new PaymentStepPage();
 
 describe("Buy a t-shirt", () => {
   it("then the t-shirt should be bought", () => {
```

Comment:
You should place the intialization of the pages with a before hook as it was shown for the dresses-list-page.spec